sentry/src/main/java/io/sentry/Scopes.java

```diff
@@ -40,10 +36,6 @@ public final class Scopes implements IScopes, MetricsApi.IMetricsInterface {
   private final @NotNull SentryOptions options;
   private volatile boolean isEnabled;
   private final @NotNull TracesSampler tracesSampler;
-
```

Comment:
Having a separate map per `Scopes` will break the feature since it's forked more often than hub used to be (at least for non mobile).